src/style.css

```diff
@@ -1 +1,44 @@
-/* add styles here */
+.page {
+  margin: 0;
+  padding: 0;
+}
+
+.page__body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url('../images/star.svg');
+  background-size: cover;
+  margin-right: 4px;
```

Comment:
I see you've reset it for the last child on the deployed version, but can't find this code here:

![image](https://github.com/user-attachments/assets/b2fde91b-fd17-4f3b-9101-cf14a04a2369)

Please, push all (latest) changes